backend/workflow_manager/workflow/workflow_helper.py

```diff
@@ -482,6 +492,7 @@ def execute_bin(
         scheduled: bool = False,
         execution_mode: Optional[tuple[str, str]] = None,
         pipeline_id: Optional[str] = None,
+        use_file_history: bool = True,
```

Comment:
NIT: will it get it from kwargs?

Reply:
This will be set and passed when the job is enqueued - [here](https://github.com/Zipstack/unstract/blob/5a36a3f6d028b3804714723b39fcc6101557654f/backend/workflow_manager/workflow/workflow_helper.py#L433)